rollup.config.js

```diff
@@ -8,13 +8,27 @@ const distDir = 'dist/js/';
 
 const pluginsES = () => [
   resolve(),
-  commonjs(),
-  process.env.NODE_ENV === 'production' ? terser({ output: { comments: false } }) : null,
+  commonjs({
+    dynamicRequireTargets: [
+      './src/_11ty/**/*.js',
```

Comment:
Just a thought: I wonder if we go a bit tighter on what gets declared here. More of an narrow/opt-in scope (i.e. this extra `dynamicRequireTargets` and `transformMixedEsModules` just to serve the 11ty shortcode ArticleCard module (and by extension the filters it relies on), because it's CommonJS?)

```js
dynamicRequireTargets: [
  './src/_11ty/filters/**/*.js',
  './src/_11ty/shortcodes/**/*.js',
],
```

There's an ongoing [discussion](https://github.com/11ty/eleventy/issues/836) about native Node support for ES modules, so hopefully this is just a temporary stopgap until we can move our 11ty scripts to the new format and drop any `module.exports`/`require` malarkey.

Reply:
Changed.